content/blog/2022-04-26-Superoptimizer.md

```diff
@@ -0,0 +1,83 @@
++++
+title = "Superoptimizer: a look at the smallest program"
```

Comment:
We usually use Title Case for these blog posts.
```suggestion
title = "Superoptimizer: A Look at the Smallest Program"
```

Reply:
Makes sense! I have changed this.